### ISSUE TITLE
Web: unable to open / focus compact menu on macOS

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#82759]
There seems to be no command that has a keybinding associated on macOS at least to trigger the compact menu. We should have a default keybinding that works on macOS (same as Linux and Windows too) to focus and show the main menu.

**Comment by isidorn** [comment]
Agree, this would be good to have.
@sbatten if you do not have time to look into this maybe you can provide a code pointer so we open this up for contributions?

**Comment by sbatten** [comment]
Right now there is no such command to trigger for this but it would not be hard to add. My main question to get this to work would be, what should the keybinding be for web? alt works in windows/linux but macOS uses ^F2 as a standard for focusing the menu bar and we cannot override it even in full screen.

**Comment by isidorn** [comment]
@sbatten thanks for looking into this!
It might be unexpected and weird for some users that we steal mac keybinding for focusing the top mac menu.
So currently I do not have a good suggestion. I just checked with our accessibility community to see if they have ideas since they know all the keybindings used for everything.

**Comment by isidorn** [comment]
How about `F10`? Is that taken?

**Comment by sbatten** [comment]
@isidorn, found out today that there is a keybinding specific to node launch configs that start debugging on f10. Should we just let this conflict and have user choose or find a new bind?

**Comment by isidorn** [comment]
@sbatten oh yeah F10 is start and stop on entry. Geeez...
I would just go with it and make sure the menu one wins. Since node-js only takes that when there is a launch configuration set.
I would maybe change the node-js one to also trigger one alt+F10
@connor4312 what do you think?

**Comment by connor4312** [comment]
I'm fine removing that as a default keybining -- js debug doesn't support it. Since no one mentioned it yet it's probably fine to have it be unbound by default and available if anyone wants it.

**Comment by isidorn** [comment]
Yeah. let's go loco and unbound by default.

**Comment by bpasero** [comment]
I am not seeing this command (using `yarn web`):

![image](https://user-images.githubusercontent.com/900690/86119732-47947980-bad3-11ea-84b1-2c9bedddd3c6.png)

**Comment by sbatten** [comment]
The keybinding was present, but it wasn't shown in the command palette. It will now show in the command palette as well.
